cypress/page/shipping-step.page.ts

```diff
@@ -4,7 +4,7 @@ class ShippingStepPage {
   private Checkout: string;
 
   constructor() {
-    this.AgreeTerms = "div.checker input";
+    this.AgreeTerms = "div.checker input[name='cgv']";
     this.Checkout = "p.cart_navigation button[name='processCarrier']";
```

Comment:
probably, just with button[name='processCarrier'] or [name='processCarrier'] should be enough

Reply:
sure !!